catalyst/dl/utils/trace.py

```diff
@@ -1,22 +1,48 @@
-from typing import TYPE_CHECKING, Union
+from typing import TYPE_CHECKING, Any, Dict, List, Callable, Union  # isort:skip
 import inspect
 from pathlib import Path
 
-import torch
 from torch import nn
-from torch.jit import ScriptModule
+from torch.jit import ScriptModule, trace, load, save
 
+from catalyst.dl import State
+from catalyst.utils.tools.typing import Device, Model
 from catalyst.utils import (
     assert_fp16_available,
-    get_fn_argsnames,
+    is_wrapped_with_ddp,
```

Comment:
`is_wrapped_with_ddp` would be deprecated, it's better to switch to `check_ddp_wrapped`